### ISSUE TITLE
Xbox Version Query 

### ISSUE THREAD

**Issue by LABWAZHERE** [schofida/CWORR#148]
As a former Xbox main, i'd just like to ask, how did you pull off the losing and switching sides aspect (i thought they required skse), 

additionally what are you using as the mod options? (For troop mults and debug stuff as an alternative to the mcm menu?)

also what is different between pc version vs Xbox 

(would like to know as I may want to use my Xbox version of Skyrim  again in the future this mod and a few others + stability reasons were the main reason i bought the pc version of skyrim)

**Comment by ds-nscorp** [comment]
Lol I was hoping that I would sneak that in without anyone noticing. I am pretty sure that I uploaded it wrong to Bethesda so I will need to upload it again tomorrow.

So I think that the CWOLeveled is going to be the more popular flavor so I mashed that into the main plug-in. Otherwise it's the same parameters as what you see when you first open the MCM on PC.

99% of this mod is plain-Jane papyrus scripting. Here are the functions that don't work:

MCM obviously

Spies. SKSE provides a really handy function to get an NPC list of factions which comes in real handy when the NPC is recycled and no longer a spy.

Boots of Springheel Jack. Rank reward. Changes a game setting parameter that allows the player to jump higher. This can only be accomplished with SKSE.

Everything else is intact.
